api/swagger.yaml

```diff
@@ -766,13 +788,20 @@ components:
 
     Data:
       type: object
+      required:
+        - current-tick
+        - planets
+        - players
```

Comment:
add player-id

Reply:
sorry, I do not see this comment in the file, but I do not have it required in purpose. IMHO it is ok not to return it when user is not logged.